website/events/templates/events/ical_help.html

```diff
@@ -0,0 +1,34 @@
+{% extends "simple_page.html" %}
+{% block page_title %}iCal feed instructions{% endblock %}
+
+{% block page_content %}
+    <p>This page provides information on how to use an iCal feed to stay updated with Thalia events.</p>
+    <h3>What is an iCal feed?</h3>
+    <p>
+        An iCal feed is a calendar format that allows you to subscribe to events and have them automatically appear in your calendar application.
+        We offer two different feeds: one with all Thalia events, and one that is personalized for you, with only the events that you've registered for.
+    </p>
+    <h3>How to subscribe to an iCal feed</h3>
+    <p>To subscribe to an iCal feed, follow these steps:</p>
+    <ol>
+        <li>Copy the iCal feed URL from below.</li>
+        <li>Open your calendar application (e.g., Google Calendar, Apple Calendar, Outlook).</li>
+        <li>
+            In your calendar application find the button to add an (iCal) calendar via its URL.
+            For example, in Google Calendar, this is "Other calendars" and then "Via URL".
+            In the macOS Calendar the option is under the "File" menu and is called "New calendar subscription".
```

Comment:
I think the double "is" is worse grammatically than single "is". The option is ((located here) and (called that)), rather than The option (is located here) and (is called that).